### ISSUE TITLE
Buttons don't have an accessible name

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#39]
Describe the bug

The icon buttons in the top bar do not have accessible label/names.

According to both Firefox console Accessibility test and Google Chrome Lighthouse, the Theme switch/toggle in top bar does not have accessible label.

"When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers"
    
To Reproduce

Run the dev server with npm start in the root dir. May need to npm i or npm install the app as I did not include my local node_modules dir. in the upload and it is in the .gitignore.

Expected behavior
Screen reader accessibility

Desktop (please complete the following information):

    Linux, x64
    Firefox, Chrome
    both 107

Additional context

**Comment by AlexOla-NG** [comment]
Hi there @gbowne1 ,

Please assign this issue to me. I would like to work on it

**Comment by gbowne1** [comment]
Ok @AlexOla-NG. I will assign you.

**Comment by AlexOla-NG** [comment]
Awesome.

Thank you👍🏾

**Comment by gbowne1** [comment]
You're wwlcome @AlexOla-NG

**Comment by AlexOla-NG** [comment]
Hi @gbowne1, I've made the changes.
Please review the PR

**Comment by gbowne1** [comment]
@AlexOla-NG thank you for the opportinity to review your PR.  I have checked it out locally, tested and merged.

**Comment by AlexOla-NG** [comment]
you're welcome, @gbowne1. I'm glad I could help out
